### PR TITLE
XKit Editor Branding

### PR DESCRIPTION
**Pull request by Wolvan**
Include New-XKit in the copyright and change the version format of the default legacy header to our new format

### CODE REVIEW

File: editor.js [hunk 1]
```diff
@@ -172,9 +172,9 @@ function extension_editor_finish_run() {
 			}
 
 			var default_script = "//* TITLE " + new_filename + " **//\n" +
-								 "//* VERSION 1.0 REV A **//\n" +
+								 "//* VERSION 1.0.0 **//\n" +
 								 "//* DESCRIPTION	**//\n" +
-								 "//* DEVELOPER STUDIOXENIX **//\n" +
+								 "//* DEVELOPER New-XKit **//\n" +
```
Comment (0xazure) [inline editor.js]: Branding question: are we 'New XKit' or 'New-XKit'?

File: editor.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
 	Extension Editor for XKit 7
 	Version 1.0
 
-	(c) 2011 - 2013 STUDIOXENIX
+	(c) 2011 - 2015 STUDIOXENIX & New-XKit Team
```
Comment (0xazure) [inline editor.js]: I would actually say

```
(c) 2011 - 2014 STUDIOXENIX
(c) 2015 New XKit Team
```

But then again, I'm no a copyright lawyer.

File: editor.js (hunk 1 above)
Comment (Wolvan) [inline editor.js]: This one is to link to our github, that's why I used new-xkit

File: editor.js (hunk 2 above)
Comment (Wolvan) [inline editor.js]: wouldn't it be 2013-2015 New XKit Team?

File: editor.js (hunk 1 above)
Comment (0xazure) [inline editor.js]: Yes, excellent point.

File: editor.js (hunk 2 above)
Comment (0xazure) [inline editor.js]: Given that https://github.com/new-xkit/XKit/commit/06b2805f76be5d4948c18811b27ac8d4f9e166fc is Atesh's last commit we have in `master` on October 18th, 2014 and https://github.com/new-xkit/XKit/commit/c4973264a0cfffe0a333c26a9b578ea5078bb753 is the first New-XKit Team commit to the repo on July 7th, 2015, I stand by my original suggestion.

Comment (Wolvan) [inline editor.js]: OK, yeah. Point taken

Comment (nightpool) [inline editor.js]: it actually should be "(c) 2015 multiple authors".

unlike studioxenix we're not one homogenous entity and can't really be
thought of as such from a copyright point of view

[IANAL, TINLA]

Comment (0xazure) [inline editor.js]: > can't really be thought of as such from a copyright point of view

Are we not releasing the software as a group ("New XKit") under the GPLv3?  We don't have an official Contributor Agreement that our contributors sign before releasing their code to us, but at this point it's kind of implied, no?

Comment (nightpool) [inline editor.js]: @0xazure [IANAL] as I understand it, each contributor individually licenses
their contributions under the gplv3, but retains their own copyright. this
leads to tricky situations wrt relicensing and dual licensing, and stuff
like that, which is why CLA are a thing. But you need some sort of legal
entity to take ownership to make a CLA work, and i don't really think its
very useful to us. (are we ever planning on relicensing away to a
non-compatible license?

Comment (0xazure) [inline editor.js]: Probably won't need to worry about relicensing, but a CLA would still be something to look into.

How about:

> (c) 2011 - 2014 STUDIOXENIX
> (c) 2015 New XKit Team and Contributors (https://github.com/new-xkit/XKit/contributors)

**Comment by finagle29** [comment]
LGTM, @homu r+

**Pull request merged**